### PR TITLE
NetCDF: bump

### PR DESCRIPTION
**Pull request by extrowerk**
base lib: bupm, lobotomized previous version
cxx: no new version available upstream
fortran: bump, nothing uses it yet, therefore no previous version kept.

### CODE REVIEW

File: sci-libs/netcdf/netcdf47-4.7.2.recipe [hunk 1]
```diff
@@ -0,0 +1,90 @@
+SUMMARY="Scientific library and interface for array oriented data access"
```
Comment (waddlesplash) [inline sci-libs/netcdf/netcdf47-4.7.2.recipe:1]: Why not name this recipe `netcdf4.7`?

Comment (Begasus) [reply sci-libs/netcdf/netcdf47-4.7.2.recipe:1]: I don't think you can have a dot in a filename (filename should be the same as in PROVIDES also).
Same as the ones for eg python, lua ...

Comment (waddlesplash) [reply sci-libs/netcdf/netcdf47-4.7.2.recipe:1]: You can, and we should here. I think PROVIDES can have dots as well.

Comment (Begasus) [reply sci-libs/netcdf/netcdf47-4.7.2.recipe:1]: Stand corrected, works fine by renaming it to netcdf4.7 (tested on 32bit nightly)

**Pull request merged**